### ISSUE TITLE
Usage of 'activity' unclear for 'active legislation', document updating concerns

### ISSUE THREAD

**Issue by emanuelfeld** [DCgov/dc-madison#62]
Difference between 'Recent Activity' (I assume that means actual changes to the document) and 'Most Active Documents' (i.e. comments and annotations).

Are documents actually being updated? What happens to the edits/annotations on previous versions? Is there the ability to tie together multiple versions?

**Comment by emanuelfeld** [comment]
Relatedly, this makes it sound like the statistics were updated 2 days ago:

![image](https://cloud.githubusercontent.com/assets/4269640/12331570/12dcb362-bab8-11e5-9562-9cba9c76c4bb.png)

**Comment by krusynth** [comment]
To answer the original issue here, `Most active` means the most comments & annotations, `Recent activity` refers to most recently updated documents (we can change this to `Recently Updated`), and `Updated` under stats refers to the document itself (we can add clarity by changing the text to `Document last updated X days ago`).

With regard to documents being updated:

The original intent of Madison is to allow user-submitted comments to shape the legislation, so yes documents are able to be edited.  In carrying out this goal, the new editor we're working on will allow admins to incorporate suggested changes directly to the documents.  Right now, however, edits simply replace the content of the currently in-place document - which is somewhat dangerous from a data standpoint.

This introduces the problem you've suggested however - comments and annotations on previous versions of a document may no longer apply.  Moreover, from a technical standpoint, it is _very_ hard to anchor annotations to a document that has changed in any way.  The solution we'd discussed internally was, If a document has an older version, show a list of older versions, and the original comments and annotations are preserved on that version.  We would ideally like to keep annotations attached to content that has not changed, but again that's technically difficult at this time.

There are _lots_ of ways to solve this problem, to be fair, and we can brainstorm on that more, but for the time being we have at least an idea of a `good enough` solution that we can always improve on later - we're not losing any data along the way.

**Comment by emanuelfeld** [comment]
Those language fixes work for me. I was thinking similarly re: keeping a list of document versions (with ability to click to see older ones with annotations still there). Difficulty with versioning and keeping annotations, is that there are annotations for only parts of lines and sometimes people highlight the whole line even if they only are commenting on a section. Hard to automatically determine if a document update addressed a concern.

**Comment by krusynth** [comment]
> Difficulty with versioning and keeping annotations, is that there are annotations for only parts of lines and sometimes people highlight the whole line even if they only are commenting on a section. Hard to automatically determine if a document update addressed a concern.

Absolutely.  The best we could do here realistically is to let users know that a thing they left an annotation on has changed in some way, and that their annotation has been archived along with the previous version, giving them the opportunity to re-comment.
